test/util.ts

```diff
@@ -9,16 +9,27 @@ import * as ts from "typescript";
 import * as vm from "vm";
 import * as tstl from "../src";
 import { createEmitOutputCollector } from "../src/transpilation/output-collector";
-import { EmitHost, getEmitOutDir, transpileProject } from "../src";
+import { EmitHost, getEmitOutDir, LuaTarget, transpileProject } from "../src";
 import { formatPathToLuaPath, normalizeSlashes } from "../src/utils";
 
-const jsonLib = fs.readFileSync(path.join(__dirname, "json.lua"), "utf8");
-const luaLib = fs.readFileSync(path.resolve(__dirname, "../dist/lualib/lualib_bundle.lua"), "utf8");
+function readLuaLib(target: tstl.LuaTarget) {
+    const luaLibDir = `../dist/${target === LuaTarget.Lua50 ? "lualib-lua50" : "lualib"}`;
```

Comment:
Also duplicated with an earlier comment I made, you can export that function and import & use it here

Reply:
Implemented with https://github.com/TypeScriptToLua/TypeScriptToLua/commit/28cc8cdd5be3ef523c9bc8905676fc02c51755e0.